applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_static_solver.py

```diff
@@ -96,27 +94,27 @@ def _update_arc_length_point_load(self, lambda_value):
             print("POINT_LOAD_Y: ", force_y)
```

Comment:
mmm

all of this of the update_arc_lenght should not be done in python ... it should be a function in c++.
Maybe a function can be created in VariableUtils for the purpose

Reply:
Will do in separate PR

Reply:
@RiccardoRossi There is only one test that uses the arc-length method, but it is outdated. I tried fixing it but didn't succeed. It seems like there is some more work necessary
Therefore I suggest doing it all together at some point, both fixing the test and implementing the function in c